src/api/status/package.json

```diff
@@ -23,12 +21,19 @@
     "@octokit/core": "3.5.1",
     "@octokit/plugin-retry": "3.0.9",
     "@octokit/plugin-throttling": "3.5.2",
+    "@popperjs/core": "2.10.2",
     "@senecacdot/satellite": "^1.x",
+    "bootstrap": "5.1.3",
     "express": "4.17.2",
     "express-handlebars": "6.0.2",
     "npm-run-all": "4.1.5",
+    "perfect-scrollbar": "1.5.1",
     "sass": "1.45.2",
-    "vite": "2.7.13"
+    "vite": "2.7.13",
```

Comment:
Is this a dev dep?

Reply:
Yes, technically. But when we need to run the build command in docker, which uses vite. Our docker only installs dependencies (which is a bit weird tbh).

Reply:
We can change the dockerfile to do this in stages, see https://github.com/Seneca-CDOT/telescope/blob/master/src/api/sso/Dockerfile for an example of how to do it in stages, and install different things in different stages.

Reply:
So I should change this line?

https://github.com/Seneca-CDOT/telescope/blob/39698783a296870d6655cd17202a30ea8df3f633/src/api/sso/Dockerfile#L32

Reply:
As a temporary solution, sure.  But the real solution here is to do this in stages.  We need the dev dependencies to do the build, so we would install them in one layer, do the build, then copy the built files to another later that only installs the production dependencies.  We can help you write this in follow-ups.

cc @TDDR, @Meneguini, @Kevan-Y.